atmo/apps.py

```diff
@@ -79,6 +79,9 @@ def ready(self):
         # https://github.com/mozilla/sugardough/issues/38
         session_csrf.monkeypatch()
 
+        # Import job signals so they are registered.
+        import atmo.jobs.signals  # noqa
```

Comment:
I know what you're doing here, but I'd rather not use imports as triggers for signal connections. Can you keep the signal callbacks in atmo.jobs.signals and connect them here instead?